packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.service.js

```diff
@@ -64,12 +64,14 @@ export default class BreadcrumbService {
             transition
               .injector(state.name)
               .getAsync('breadcrumbPrefix')
-              .then(({ name, url }) => {
-                this.breadcrumb.unshift({
-                  name,
-                  url,
-                  active: false,
-                });
+              .then((prefixs) => {
+                prefixs.forEach(({ name, url }, index, prefixArray) =>
+                  this.breadcrumb.splice(index, 0, {
```

Comment:
**suggestion:** Use `unshift` method to make it uniform with above implementation which will result in better readability

Reply:
if I use unshift, I will need to write prefixes.reverse().forEach to be sure the breadcrumb is in the right place. You confirm this change ?

Reply:
Since it was a suggestion, I let you decide and I am okay with any of the implementations